.github/workflows/ci-additional.yaml

```diff
@@ -0,0 +1,188 @@
+name: CI Additional
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+  workflow_dispatch: # allows you to trigger manually
+
+jobs:
+  detect-ci-trigger:
+    name: detect ci trigger
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' || github.event_name == 'pull_request'
+    outputs:
+      triggered: ${{ steps.detect-trigger.outputs.trigger-found }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/detect-ci-trigger
```

Comment:
This action was moved into its own repository. 

You can replace `uses: ./.github/actions/detect-ci-trigger` with 

```yaml
uses: xarray-contrib/ci-trigger@v1
```